VRPB/README.md

```diff
@@ -0,0 +1,10 @@
+# VRPB
+
+VRP with backhaul instances of Queiroga et al. (2020).
+Downloaded from [VRP-REP](http://www.vrp-rep.org/references/item/queiroga-et-al-2019.html) and modified to fit the VRPLIB format. 
+This repository only includes the 90 largest VRPB instances, which we use for benchmarking PyVRP.
+The other instances can be found in [this zip](https://github.com/PyVRP/PyVRP/files/14402701/VRPB.zip).
```

Comment:
Which variants? I see three `n1001`, but what's the difference?

Reply:
Maybe say something about "there are three variants of the VRPB problem: <list>. We include the 30 largest instances of Queiroga et al. (2020) of each variant in this repository."